.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Prerequisites
+
+- [ ] Have you written a descriptive issue title?
+- [ ] Have you verified that you are running the latest version of Nancy?
+- [ ] Have you verified if the problem exist in both `debug` and `release` mode?
+- [ ] Have you searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported?
+
+### Description
+[A description of the bug or feature]

```

Comment:
@khellang you are talking about HTML5 placeholders, right? How would we do that? This is just a markdown file that gets entered into the text field when you create a new issue/pull-request. I checked a whole bunch of other repos how they did it and this seemed to be the general approach.

I kind of like it being actual text as well, it will force people to actively replace it :D